flink-connectors/flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/table/JdbcDynamicTableSinkITCase.java

```diff
@@ -316,4 +338,39 @@ public void testReadingFromChangelogSource() throws Exception {
 			Row.of("user4", "Tina", "tina@gmail.com", new BigDecimal("11.30"), new BigDecimal("22.60"))
 		}, DB_URL, USER_TABLE, new String[]{"user_id", "user_name", "email", "balance", "balance2"});
 	}
+
+	@Test
+	public void testCheckpoint() throws Exception {
```

Comment:
Rename to `testFlushBufferWhenCheckpoint` to make the test purpose more explicitly.